web-frontend/src/layouts/LandingPage/LandingPage.tsx

```diff
@@ -9,7 +9,10 @@ interface LandingPageProps {
 
 export default function LandingPage(props: LandingPageProps) {
   return (
-    <div className="bg-brand bg-landing-page bg-no-repeat bg-cover bg-center min-h-screen">
+    <div
+      className={`bg-brand bg-landing-page bg-no-repeat bg-cover
```

Comment:
if you change the landing page here than some pages will need to be reformatted... maybe it's worth making a separate issue  for the update ?

Reply:
Done